packages/editor-ui/src/plugins/i18n/locales/en.json

```diff
@@ -1114,6 +1113,40 @@
 	"workflowSettings.timeoutAfter": "Timeout After",
 	"workflowSettings.timeoutWorkflow": "Timeout Workflow",
 	"workflowSettings.timezone": "Timezone",
+	"workflows.heading": "Workflows",
+	"workflows.add": "Add Workflow",
+	"workflows.menu.my": "My workflows",
+	"workflows.menu.all": "All workflows",
+	"workflows.item.open": "Open",
+	"workflows.item.delete": "Delete",
+	"workflows.item.updated": "Last updated",
+	"workflows.item.created": "Created",
+	"workflows.item.owner": "Owner",
+	"workflows.item.active": "Active",
+	"workflows.item.inactive": "Inactive",
+	"workflows.search.placeholder": "Search workflows...",
+	"workflows.filters": "Filters",
+	"workflows.filters.tags": "Tags",
+	"workflows.filters.ownedBy": "Owned by",
+	"workflows.filters.sharedWith": "Shared with",
+	"workflows.filters.apply": "Apply filters",
+	"workflows.filters.reset": "Reset all",
+	"workflows.filters.active": "Some workflows may be hidden since filters are applied.",
+	"workflows.filters.active.reset": "Remove filters",
+	"workflows.sort.lastUpdated": "Sort by last updated",
+	"workflows.sort.lastCreated": "Sort by last created",
+	"workflows.sort.nameAsc": "Sort by name (A-Z)",
+	"workflows.sort.nameDesc": "Sort by name (Z-A)",
```

Comment:
a few of these don't seem to be used

Reply:
Most of them will be needed for the workflow sharing feature. I'll add a reminder to remove what's not needed afterwards if that's ok with you.